pkg/pow/solutionHandler.go

```diff
@@ -54,20 +58,37 @@ func (s *SolutionHandler) Submit(ctx context.Context, result *Result) (*types.Tr
 			return nil, errors.Wrapf(err, "could not retrieve pricing data for current request id")
 		}
 		val := m[valKey]
-		if len(val) == 0 {
-			s.log.Warn("have not retrieved price data for requestId %d. WARNING: Submitting 0 because of faulty API request", challenge.RequestIDs[i].Uint64())
-		}
-		value, err := hexutil.DecodeBig(string(val))
-		if err != nil {
-			if challenge.RequestIDs[i].Uint64() > 53 {
-				s.log.Error("problem decoding price value prior to submitting solution: %v\n", err)
-				if len(val) == 0 {
-					s.log.Error("0 value being submitted")
-					s.currentValues[i] = big.NewInt(0)
+		value := big.NewInt(0)
```

Comment:
lines like this cost me big before, maybe this should be removed too? Not sure it's needed since you have lines 78 and 80 here, maybe it is though